### DISCUSSION TITLE
A security problem on hidden accordions

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12754] [Archive]
Accordions is a nice feature of Directus UI.
In a form, I have some accordions that must be hidden for the average user.

To implement this, I Edit the accordion field to set a condition filter that hides it for the average user. It works, but

1. The title of the accordion is still visible
2. The body of the accordion is just set to `display: none` by Directus. For a seasoned user it is easy to hit F12 and change it to `display: block` in order to see all the information inside the accordion he should not see.

The fields hidden directly are not sent to the end user.
Could we have this with accordions?

Just to add that the same problem appears if we just set the accordion to `Hide Field on Detail` from the  ⋮  menu

**Comment by rijkvanzanten** [comment]
Could you share the exact configuration of your fields? 

As for the display question, Hidden is purely meant for form layouting, and isn't supposed to be a security measure. If you want to hide fields from a user completely, use permissions instead. That will remove the field, and prevent the user from accessing any of the content 👍🏻

**Reply by Pachat** [reply]
Yes, I saw that hiding the fields in the accordion one by one would not let them appear.

That said, it would be nice when the accordion itself is set to hidden not to show its title
\> Title of an accordion
~~\> Title of the hidden accordion~~
\> Title of another accordion
